README.md

```diff
@@ -1155,6 +1155,24 @@ every { quit(1) } throws Exception("this is a test")
 * clear - deletes the internal state of objects associated with a mock, resulting in an empty object
 * unmock - re-assigns transformation of classes back to original state prior to mock
 
+### Scoped mock
+
+Scoped mock is a mock that unmocks it after the block has been executed.
```

Comment:
"A scoped mock is a mock that automatically unmocks itself after the code block passed as a parameter has been executed".